packages/icon/src/index.js

```diff
@@ -0,0 +1 @@
+export { default, AVAILABLE_NAMES, names, sizes } from './Icon';
```

Comment:
We should export `AVAILABLE_NAMES`, `names`, and `sizes` from their respective modules, instead of passing them through the `Icon` module.